src/inc/debugmacros.h

```diff
@@ -29,6 +30,7 @@ bool GetStackTraceAtContext(SString & s, struct _CONTEXT * pContext);
 void _cdecl DbgWriteEx(LPCTSTR szFmt, ...);
 bool _DbgBreakCheck(LPCSTR szFile, int iLine, LPCSTR szExpr, BOOL fConstrained = FALSE);
 
+ANALYZER_NORETURN
```

Comment:
GCC likes extern at the beginning of the line.

Reply:
Ok, let me change that so that you don't have to patch it afterwards.

Reply:
thanks a lot